### ISSUE TITLE
Warning: Task "scsslint:allFiles" failed. Use --force to continue.

### ISSUE THREAD

**Issue by douglasdeodato** [ahmednuaman/grunt-scss-lint#133]
i am getting this warning above this is my gruntfile.js

`    scsslint: {
        allFiles: [
          'scss/_base.scss',
        ],
        options: {
          config: 'scss/.scss-lint.yml',
          reporterOutput: 'scss/scss-lint-report.xml',
          colorizeOutput: true,
          compact:false
        }
      },`

error:

Running scss-lint on allFiles
scss/core/_base.scss:1 [W] Comment: Use `//` comments everywhere
scss/core/_base.scss:8 [W] TrailingWhitespace: Line contains trailing whitespace
scss/core/_base.scss:13 [W] PropertySortOrder: Properties should be ordered background-color, font, height, min-height, width
scss/core/_base.scss:16 [W] NameFormat: Name of variable `darkWhite` should be written in all lowercase letters with hyphens instead of underscores
scss/core/_base.scss:23 [W] PropertySortOrder: Properties should be ordered list-style, margin, margin-bottom, padding
scss/core/_base.scss:40 [W] NameFormat: Name of mixin `remSize` should be written in all lowercase letters with hyphens instead of underscores
Results have been written to: scss/scss-lint-report.xml
Warning: Task "scsslint:allFiles" failed. Use --force to continue.

Aborted due to warnings.

scss-lint-report.xml is created in my folder showing the erros, how can i remove this message : Warning: Task "scsslint:allFiles" failed. Use --force to continue.

when my file is fixed i get:

Running scss-lint on allFiles
>> 1 file is lint free
Results have been written to: scss/scss-lint-report.xml

Done, without errors.

**Comment by jshthornton** [comment]
I agree on this one. The grunt runner should take the severity from scss-lint and determine whether it was successful (albeit with warnings) or unsuccessful due to errors.

**Comment by douglasdeodato** [comment]
make it a bit confuse,special for people with no knowledge.

instead: Warning: Task "scsslint:allFiles" failed. Use --force to continue.

Aborted due to warnings.


why not: scsslint Task finished, please check the errors above to have your file with a correct syntax. 

or something like that no? make more sense then task failed --force to continue.

usually task failed --force to continue appears when the gruntfile.js has a syntax  error .

anyway my opnion.

**Comment by ahmednuaman** [comment]
PR?

**Comment by douglasdeodato** [comment]
+1

**Comment by douglasdeodato** [comment]
oh Pull Request :) , I will try.

**Comment by ahmednuaman** [comment]
👍

**Comment by nick11703** [comment]
I think PR #138 will help with this.

**Comment by douglasdeodato** [comment]
thank you @nick11703   @ahmednuaman  ?  :)

**Comment by douglasdeodato** [comment]
@nick11703  and @ahmednuaman  any news about this pull request? thank you.

**Comment by nick11703** [comment]
We need the project owner @ahmednuaman to approve and merge PR #138
